hack/code/bandwidth_gen/main.go

```diff
@@ -70,13 +71,14 @@ func main() {
 
 			doc := lo.Must(goquery.NewDocumentFromReader(response.Body))
 
-			// grab two tables that contain the network performance values
-			// first table will contain all the instance type and bandwidth data
-			// some rows will will have vague describe such as "Very Low", "Low", "Low to Moderate", etc.
-			// These instance types will can be found on the second table with absolute values in Gbps
-			// If the instance type is skipped on the first table it will be grabbed on the second table
+			// grab the table that contains the network performance values
+			// some rows will have vague description such as "Very Low", "Low", "Low to Moderate", etc.
```

Comment:
Also, would it make sense to have a list of instance types we expect not to have any network info. That way we don't need to include them in the genrated file